pkg/ExternalPackages/versions.props

```diff
@@ -2,19 +2,28 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
   <ItemGroup>
-    <ExternalPackage Include="runtime.win7-amd64.runtime.native.System.Data.SqlClient.sni">
+    <ExternalPackage Include="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni">
```

Comment:
Correct, "ExternalPackage" is a signal to building tests against packages to not update those packages.  Eric's statement is correct though ("no-one takes a direct ref to these packages from project.json"), so I don't think it's actually utilized.  Given that there is no direct reference to these packages in the test's project.json, any locking of package versions for these runtime packages so that they are consuming a different runtime package version than what is in identity package, must occur somewhere during packaging and not during building tests against packages.

I don't think anyone cares about these.